li-apache-kafka-clients/src/main/java/com/linkedin/kafka/clients/consumer/LiKafkaConsumer.java

```diff
@@ -168,10 +171,35 @@
    * partitions of that topic.
    *
    * @param pattern Pattern to subscribe to
+   * @param callback listener instance to get notifications on partition assignment/revocation for the
+   *                 subscribed topics
    */
   @InterfaceOrigin.ApacheKafka
   void subscribe(Pattern pattern, ConsumerRebalanceListener callback);
 
+  /**
+   * Subscribe to all topics matching specified pattern to get dynamically assigned partitions. The pattern matching will
```

Comment:
Is this just copied from Kakfa?  Maybe we should not bother with copying the docs from Kafka open source?

Reply:
Used `{@inheritDoc}` to avoid duplication. Comments about large message support have been kept.